bleach/sanitizer.py

```diff
@@ -593,7 +593,8 @@ def sanitize_css(self, style):
         # the whole thing.
         parts = style.split(';')
         gauntlet = re.compile(
-            r"""^([-/:,#%.'"\sa-zA-Z0-9!]|\w-\w|'[\s\w]+'\s*|"[\s\w]+"|\([\d,%\.\s]+\))*$"""
+            r"""^([-/:,#%.'"\sa-zA-Z0-9!\w]|\w-\w|'[\s\w]+'\s*|"[\s\w]+"|\([\d,%\.\s]+\))*$""",
```

Comment:
For some backstory, this gauntlet regex is mediocre and I'm not even sure I understand what it's really doing. Is it enforcing valid declaration structure? Doesn't seem like it is. It originally came from html5lib and I think it's been tweaked periodically since then, but I'd bet it doesn't correctly match CSS declarations. Issue #248 covers replacing this with tinycss or tinycss2. I started looking into that, but then hit some issues and stopped.

Looks like you added `\w` which will match word characters and `_` in that first part. I think that's fine, but I wonder why it wasn't like that in the first place. Regardless, I think you can remove the `a-zA-Z0-9` and `-` in the first part. So then the regex is like this:

```
             r"""^([/:,#%.'"\s!\w]|\w-\w|'[\s\w]+'\s*|"[\s\w]+"|\([\d,%\.\s]+\))*$""", 
```

Reply:
done in 33f294d